docs/guides/build-your-first-app.mdx

```diff
@@ -10,7 +10,7 @@ You have now [rendered your first scene](./render-your-first-scene.mdx) and [imp
 
 We'll be using the Vertex Web UI Components, which are standards-based [web components](https://developer.mozilla.org/en-US/docs/Web/Web_Components). You could create your app using plain JavaScript, but we'll use [NextJS](https://nextjs.org/), a [React](https://reactjs.org/)-based framework with a great developer experience.
 
-To get an idea of what you'll be building, check out this [example application](https://nextjs-starter.vertexvis.io/?clientId=08F675C4AACE8C0214362DB5EFD4FACAFA556D463ECA00877CB225157EF58BFA&streamKey=U9cSWVb7fvS9k-NQcT28uZG6wtm6xmiG0ctU) we created.
+To get an idea of what you'll be building, check out this [example application](https://nextjs-starter.vertexvis.io/) we created.
```

Comment:
The app itself handles defaulting this stuff. That way, we can change the stream key without updating this link.